### ISSUE TITLE
Coherent: Header Image is not working

### ISSUE THREAD

**Issue by Gustavo-Hilario** [Automattic/themes#5891]
### Quick summary

When setting a `header image`, the header image isn't showing up either on the site's customizer and home page.

### Steps to reproduce

1. Activate the `Coherent` theme.
2. Go to WordPress.com > My Sites > Appearance > Customize > Header Image.
3. Upload an image or try to use one of the suggested images.

No header image will appear there.

### What you expected to happen

The header image to show up on the site's header.

### What actually happened

The header image is missing.

### Context

Report here: 4951496-zen

### Simple, Atomic or both?

Simple, Atomic

### Theme-specific issue?

Coherent

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

Yes.

### Workaround details

Changing themes.

**Comment by Robertght** [comment]
It looks like this may be a thing specific for the posts page, but I'm not 100% sure;

>If you have a custom header image, it appears full-width on the posts page, creating an impact right away.

https://wordpress.com/theme/coherent/

**Comment by kerrynicl** [comment]
I checked here and i'm not able to see the header on the site or customizer either
<img width="957" alt="Markup 2022-04-19 at 11 16 45" src="https://user-images.githubusercontent.com/83325600/164049614-81aae16b-9964-40b2-b084-dd1630b351d8.png">

**Comment by Gustavo-Hilario** [comment]
Yep. I noticed that from the theme doc. However, the header image is still not showing up on the `blog page`. It's not working either on the customizer or published page.

**Comment by Robertght** [comment]
The Header image appears to work only if the homepage is set to display the Latest Posts.

**Comment by Gustavo-Hilario** [comment]
Yes! I can confirm that. Maybe this is a `bug` per si, but we should at least clarify that on the theme docs. It would be even better if we had a notice on Customizer when this happens.
